integration_test/integration_test.go

```diff
@@ -298,6 +351,621 @@ func (t *WASMIntegrationTestSuite) Test04_ContractExecution() {
 	assert.Equal(t.T(), "50000000", amtResp.Balance)
 }
 
+func (t *WASMIntegrationTestSuite) Test12_GeneralVoluntaryValidatorRegistryUnregistryDelegation() {
+	amt := sdktypes.NewInt(1000000000000000000)
+
+	{
+		fmt.Println("Preparing proposal to add a voluntary validator")
```

Comment:
Voluntary -> Volunteer

Reply:
Volunteer (O)
Volunteer validator (X)
Voluntary validator (O)

Reply:
We decided to be volunteer validators.

Reply:
https://github.com/xpladev/xpla/pull/84/commits/768876be2661a489d6945bb3deb2448e78681862